common/src/main/java/com/jobexecution/service/exception/BadRequestException.java

```diff
@@ -0,0 +1,24 @@
+package com.jobexecution.service.exception;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
```

Comment:
Remove this

Reply:
@piyushsoni21 What needs to be removed?
`@Getter` annotation?